cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -83,9 +83,14 @@ message Options {
 	// This currently only affects LCOW. The sandbox containers scratch space is re-used for all
 	// subsequent containers launched in the pod.  
 	bool share_scratch = 14;
+	
 	//NCProxyAddr is the address of the network configuration proxy service. If omitted 
 	// the network is setup locally. 
 	string NCProxyAddr = 15; 
+
+	// job_container specifies whether to have the runtime create job containers 
+	// instead of the standard hcs containers. 
+	bool job_container = 16;
```

Comment:
Do we not want a new runtime type for this container type?

Reply:
We'd decided against it, as there will be a field on the cri spec/user facing k8s specs itself to specify for this anyways.

Reply:
Also leaves it so that if there's ever a "privileged" like container in the UVM in the future you can just swap between the hypervisor/process runtimes and just flip the `HostProcess` bool on the cri spec

Reply:
I don't understand what this option does. Does it make every container created be a job container, regardless of annotations/OCI spec passed? Shouldn't we just rely on annotations/OCI instead?

Reply:
Yea come to think of it, I think this was just for Marks testing originally

Reply:
This can be removed now

Reply:
Oop, nope